src/PlusDataCollection/Andor/vtkPlusAndorVideoSource.h

```diff
@@ -73,6 +73,9 @@ class vtkPlusDataCollectionExport vtkPlusAndorVideoSource: public vtkPlusDevice
   /*! Set horizontal shift speed. */
   PlusStatus SetHSSpeed(int type, int index);
```

Comment:
No renaming this to *Index?

Reply:
The framework that relies on the PlusConfig xml for setting values on startup (HSSpeed="0 1" will call SetHSSpeed(0, 1)) was the reason I didn't change the name here because then the config file would say have to say `HSSpeedIndex="0 1"` and only one of those values in the array is the actual index. The first value is actually the type of output amplification.